README.md

```diff
@@ -30,3 +24,143 @@ We then start a consumer with the corresponding configurations to decrypt the fi
 
 In order to have a realistic scenario, we do not produce and consume via the CLI but develop a
 producer and consumer application with Kotlin.
+
+## Create Tag
+
+We first need to create a tag on which we apply the encryption later, such as `PII`.
+As of today, we need to create the tag in the Stream Catalog first, see the [documentation](https://docs.confluent.io/platform/current/schema-registry/fundamentals/data-contracts.html#tags) of Data Contracts.
+
+## AWS KMS
+
+In the KMS section of the AWS Management Console, create a new Symmetric Key with Encrypt/Decrypt configuration
+
+![](images/aws_create_key.jpg)
+
+![](images/aws_key_config.jpg)
+
+As you click through this process you will be asked to define `Admins` and `Users` for your key. Ensure you grant access to the `User` that you want use in your Producer/Consumer app.
+
+![](images/aws_key_users.jpg)
+
+## AWS IAM
+
+After your Key has been created, navigate to AWS IAM and create an Access Key for the User that you granted permissions to in the previous step.
+
+![](images/aws_create_access_key.jpg)
+
+:warning: **Important:** Ensure you copy your Access Key ID and Secret (or download the csv file) :warning:
+
+## Update the Producer and Consumer Properties Files
+
+Update the `rule.executors._default_.param.access.key.id` and `rule.executors._default_.param.secret.access.key` properties for the Producer and Consumer with the Access Key and Secret you retrieved in the previous step.
```

Comment:
This part is a duplicate of the properties later on. Is it required or can be removed?